zebedee-cms/src/main/java/com/github/onsdigital/zebedee/model/publishing/Publisher.java

```diff
@@ -772,6 +777,12 @@ private static void sendToKafka(Collection collection) throws IOException {
         }
     }
 
+    public static String convertUriForEvent(String uri){
```

Comment:
Does this method need to be public? If so add some JavaDoc to explain this method. Alternatively could it be private?

Reply:
adding javadoc because the unit test is on this method so keeping it as public

Reply:
If it's only for accessibility by the unit test, you could resolve this by making it `protected` instead of `public`. I don't mind it being public if there is good reason for it to be made available to other packages but I can't see what that is. I won't object though as I see other methods in the class are also public when they may well have been better as private or protected.